src/meta/processors/BaseProcessor.inl

```diff
@@ -454,5 +454,21 @@ BaseProcessor<RESP>::indexCheck(const std::vector<nebula::cpp2::IndexItem>& item
     return cpp2::ErrorCode::SUCCEEDED;
 }
 
+template<typename RESP>
```

Comment:
I think better put this function to `processor/indexMan/` folder, because it's not the concept of `BaseProcessor`.

Reply:
Please check `item.get_fields().szie() >= fields.size()`.

Reply:
before we check the index have checked the fields size.